### PR TITLE
Dropdown and Panel: Fluent Styling updates to match current Fabric design specs.

### LINKED ISSUE #9305: Dropdown: Fluent Updates
- [x] Update component current hover state (gray110) to reflect toolkit (cyanBlue10) -> double check with Ben

- [x] Update component current focus state(gray160) to reflect toolkit (black) -> double check with Ben

### PR DESCRIPTION
**Pull request by marygans**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #9305 and #7920 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

This is updating the Dropdown border colors on focus and hover in fluent-theme package and `Panel.styles.ts` padding, margins, and footer alignment to match current Fabric design specs. 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9337)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts [hunk 1]
```diff
@@ -159,8 +159,8 @@ const sharedPaddingStyles = {
   paddingRight: '16px',
   selectors: {
     [`@media screen and (min-width: ${ScreenWidthMinLarge}px)`]: {
-      paddingLeft: '32px',
-      paddingRight: '32px'
+      paddingLeft: '15px',
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: Why are this changes to `Panel` inside the default styles in `office-ui-fabric-react` package? Shouldn't these be in the `fluent-theme` package?

File: packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts [hunk 2]
```diff
@@ -243,7 +243,7 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
       classNames.main,
       {
         backgroundColor: palette.white,
-        boxShadow: '0px 0px 30px 0px rgba(0,0,0,0.2)',
+        boxShadow: '0px 25.6px 57.6px 0px rgba(0, 0, 0, 0.22), 0px 4.8px 14.4px 0px rgba(0, 0, 0, 0.18)',
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: Same: these are fluent shadows that are not supposed to be in the default styles for v6.0.

File: packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts (hunk 1 above)
Comment (marygans) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: You're right about that, sorry! Working on updating now.

**Comment by leddie24** [comment]
Closing PR, we're holding off on Fluent changes until after Fabric 7.

**Pull request closed without merging**